diagnostic.md

```diff
@@ -13,57 +14,53 @@ Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
+$('#OL.children()')
+$('#OL.children().#li')
```

Comment:
your calling everything inside the parentheses here. it should be `$('ol').children()` or you could do  `$('ol li')`